front/app/templates/application.hbs

```diff
@@ -1,10 +1,12 @@
-<a href="https://github.com/dikaiosune/rust-dashboard" target="_blank">
-  <img style="position: absolute; top: 0; left: 0; border: 0; z-index: 2000"
-       alt="Fork me on GitHub"
-       src="https://s3.amazonaws.com/github/ribbons/forkme_left_darkblue_121621.png">
-</a>
+<a class="github-fork-ribbon left-top" href="https://github.com/dikaiosune/rust-dashboard" title="Fork me on GitHub">Fork me on GitHub</a> 
 
 <nav class="container">
+    {{#if isBrowserOutdated}}
+        <div style="padding: 20px 0;background: black;color: red;text-align: center;">
+            Sorry your browser is outdated. Please update
```

Comment:
I think we probably want to provide information to the user about how they might update, right? See my other comment about relying on an external service for this.